ui/cypress/e2e/tasks.test.ts

```diff
@@ -61,80 +61,64 @@ from(bucket: "${name}")
       .and('contain', taskName)
   })
 
-  it('can delete a task', () => {
-    cy.get('@org').then(({id}: Organization) => {
-      cy.get<string>('@token').then(token => {
-        cy.createTask(token, id)
-        cy.createTask(token, id)
-      })
-
-      cy.fixture('routes').then(({orgs}) => {
-        cy.visit(`${orgs}/${id}/tasks`)
-      })
-
-      cy.getByTestID('task-card').should('have.length', 2)
-
-      cy.getByTestID('task-card')
-        .first()
-        .trigger('mouseover')
-        .within(() => {
-          cy.getByTestID('context-delete-menu').click()
-          cy.getByTestID('context-delete-task').click()
+  describe('When tasks already exsist', () => {
```

Comment:
exists?

Reply:
lol @zoesteinkamp ☝️  😂 

good catch @ebb-tide ! we had a little laugh about Zoe's spelling abilities during our pairing session